pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -337,3 +341,73 @@ class DualRK45PseudoStepper(DualRKVdH2RPseudoStepper):
     @property
     def _pseudo_stepper_order(self):
         return 4
+
+def _get_coefficients(scheme):
+        a = []
+
+        scheme = re.sub(r'(?<=\))\s*,?\s*(?!$)', r'\n', scheme)
+        for l in scheme.splitlines():
+            if not l:
+                continue
+
+            a.append([float(f) for f in l.split()])
+
+        # The last row is the b vector
+        return a[:-1], a[-1]
+
+
+class DualDenseRKPseudoStepper(BaseDualPseudoStepper):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        scheme = resource_string(
+            __name__, '{}/{}'.format('densepseudoschemes/schemes', self.path)
+        )
+
+        self.a, self.b = _get_coefficients(scheme.decode('utf-8'))
+
+    @property
+    def _pseudo_stepper_has_lerrest(self):
+        return False
+
+    @property
+    def _stepper_nfevals(self):
+        return self.nstages
+
+    @property
+    def _pseudo_stepper_order(self):
+        return 1
+
+    @property
+    def _pseudo_stepper_nregs(self):
+        return self.nstages + 1
+
+    def step(self, t):
+        add = self._add
+        rhs = self._rhs_with_dts
+
+        r = self._pseudo_stepper_regidx
+
+        # Ensure r0 references the bank containing u(n+1,m)
+        if r[0] != self._idxcurr:
+            r[0], r[1] = r[1], r[0]
+
+        # First stage
+        rhs(t, r[0], r[1])
+
+        # Pre-multiply a matrix and b vector with dtau
+        a = [[self._dtau*j for j in i] for i in self.a]
+        b = [self._dtau*i for i in self.b]
```

Comment:
These are floats so maybe ar is better than i and bi is better than i.

Reply:
I missed this. Will push again soon.